test/passing/tests/class_sig-after.mli.ref

```diff
@@ -22,10 +22,22 @@ class c : object
   (** Standalone doc-string. *)
 end
 
-class unix_mockup : foooo:string -> foooo:string -> foooo:string ->
-  foooo:string -> foooo:string -> foooo:string -> bar
-
-class unix_mockup : foooo:string -> foooo:string -> foooo:string ->
-  foooo:string -> foooo:string -> foooo:string -> object
+class unix_mockup :
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  bar
+
+class unix_mockup :
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  foooo:string ->
+  object
   method foo : string
 end
```

Comment:
With the new test case we can see that the indentation of the `object` keyword isn't so nice as it aligns with both the argument before and the content of the object.

It would be very bad to indent the whole object here as that would completely re-indent the object if an argument is added.

Perhaps the `object` keyword could be exceptionally de-indented ?
```ocaml
class unix_mockup :
  foooo:string ->
  foooo:string ->
  foooo:string ->
  foooo:string ->
  foooo:string ->
  foooo:string ->
object
  method foo : string
end
```

That's not blocking this PR though, as the object is formatted the same as before.

Reply:
For the same reason, I reverted the change I made to the indentation after a class types `let open`. I've opened an issue for this: https://github.com/ocaml-ppx/ocamlformat/issues/2423

Reply:
It wouldn't shock me to de-indent the `object` keyword, that's done for `struct` in some cases and it looks nice:
```ocaml
module M (A : sig
  module type T
end) (B : sig
  module type T
end) =
struct
  let f : ((module A.T), (module B.T)) t -> string = function B s -> s
end
```